superset/assets/src/SqlLab/main.less

```diff
@@ -251,39 +252,39 @@ div.Workspace {
         padding-left: 10px;
     }
 
-    .schemaPane-enter-done, .schemaPane-exit {
-        transform: translateX(0);
-    }
+    // .schemaPane-enter-done, .schemaPane-exit {
```

Comment:
Can we delete instead of commenting?

Reply:
I just wasn’t 100% sure these are not used, but sure... I’ll nuke ‘em, and we can rebuild from git history if needed.